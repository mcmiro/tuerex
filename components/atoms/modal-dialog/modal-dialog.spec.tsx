import * as React from "react";
import {screen, render} from "@testing-library/react";
import ModalDialog from "./";

describe("Test ModalDialog component", () => {
	it("Should pass when children in ModalDialog is defined", () => {
		render(<ModalDialog data-testid="modal-test">Test</ModalDialog>);
		const megaMenu = screen.queryByTestId("modal-test");
		expect(megaMenu).toBeDefined();
		expect(megaMenu).toMatchSnapshot();
	});
});
