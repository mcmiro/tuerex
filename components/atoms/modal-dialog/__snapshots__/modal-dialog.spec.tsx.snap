// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ModalDialog component Should pass when children in ModalDialog is defined 1`] = `null`;
